raiden/tests/unit/transfer/test_node.py

```diff
@@ -40,4 +43,22 @@ def test_is_transaction_effect_satisfied(
     # finally call with us being the participant and not the partner which should check out
     state_change.participant = netting_channel_state.partner_state.address
     state_change.partner = netting_channel_state.our_state.address
-    assert is_transaction_effect_satisfied(chain_state, transaction, state_change)
+
+    # ContractSendChannelBatchUnlock would only be satisfied if both sides are unlocked
```

Comment:
so perhaps assert that the effect is not satisfied here?

Reply:
done